### PR TITLE
Use per-service 'test:<fwk>' github label instead of default 'test:shakedown' each time

### PR DESCRIPTION
**Pull request by nickbp**
At the moment, each successive test run hides the result of prior runs in github

### CODE REVIEW

**Comment by loren** [comment]
Great idea! I'm seeing it as `test:test:<fwk>` though.
screen shot 2017-02-17 at 1 32 35 pm

**Comment by nickbp** [comment]
Oh whoops, fixed

File: test.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ function run_framework_tests {
     fi
 
     # Run shakedown tests in framework directory:
-    TEST_GITHUB_LABEL="test:${framework}" ${REPO_ROOT_DIR}/tools/run_tests.py shakedown ${FRAMEWORK_DIR}/tests/
+    TEST_GITHUB_LABEL="${framework}" ${REPO_ROOT_DIR}/tools/run_tests.py shakedown ${FRAMEWORK_DIR}/tests/
```
Comment (loren) [inline test.sh]: nit: ditch the quotes?

Comment (nickbp) [inline test.sh]: wouldn't expect framework dirs to have spaces, but just in case...

**Pull request merged**